### ISSUE TITLE
Accessible tabs: Consider implementation recommendations and review with DAC

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2371]
Consider all suggestions listed here: https://inclusive-components.design/tabbed-interfaces/ and confirm/review with DAC

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/issues/1690#issuecomment-1204963513_

**Comment by daniellecorke** [comment]
On the UCD Board: https://jira.ons.gov.uk/browse/CU-92.

**Comment by daniellecorke** [comment]
Closing - this ticket cannot be prioritised for DAC and did not fail the recent DAC assessment.
